shared/javascripts/network_service.js

```diff
@@ -446,6 +446,7 @@ var privlyNetworkService = {
     $(".logged_out_nav").show();
     $(".injected_hide").show();
     privlyNetworkService.mobileHide();
+    androidJsBridge.showLoginActivity();
```

Comment:
This will take care of the platform in which the application is running.
var ua = navigator.userAgent.toLowerCase();
    var isAndroid = ua.indexOf("android") > -1; //&& ua.indexOf("mobile");
    if(isAndroid) {
     androidJsBridge.showLoginActivity();
    }